jax/experimental/stax.py

```diff
@@ -260,6 +260,22 @@ def apply_fun(params, inputs, **kwargs):
   return init_fun, apply_fun
 
 
+def ResBlock(*layers, fan_in=FanInSum, tail=Relu):
```

Comment:
You can’t use this syntax yet because JAX still supports Python 2 (for a little while longer)

Reply:
> Looks reasonable to me, though my preference would be to avoid the final “tail” layer defaulting to ReLU and keep all activations explicit. This would be more consistent with the rest of stax, and in my experience unexpected activations are a common source of bugs.

Oh interesting--in hindsight I could see that potentially being an issue but I don't think I would have thought of it before you mentioned it.

>You can’t use this syntax yet because JAX still supports Python 2 (for a little while longer)

I submitted a patch that works for Python 2 and 3 with a `TODO` to replace it with the more readable function head when Python 2 support is dropped.

Please feel free to let me know if you would like any further modifications.